airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -473,11 +482,11 @@ def create_external_table(self,
     def run_query(self,
                   bql=None,
                   sql=None,
-                  destination_dataset_table=False,
+                  destination_dataset_table=None,
                   write_disposition='WRITE_EMPTY',
                   allow_large_results=False,
-                  flatten_results=None,
-                  udf_config=False,
+                  flatten_results=False,
```

Comment:
This breaks the BigQueryOperator which sets the udf_config to false when it calls the hook.

Reply:
@draksler, very sad what is was not covered by tests in BigQueryOperator and it not matched docstrign, because udf_config it has a list type, not bool.. need to open the bug issue and make a fix

Reply:
@draksler Thanks for reporting. Will fix this bug and release it in 1.10.2

Reply:
@xnuinside If you want to work on this, can you please make sure to add tests to cover this cases.

Reply:
I've opened a jira [here](https://issues.apache.org/jira/browse/AIRFLOW-3438), if it is a duplicate please close it. 
Tests would normally not cover this case, as you would mock the whole hook, not part of it. It is unfortunate that the docstring was incorrect though.